.vscode/launch.json

```diff
@@ -5,6 +5,23 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
if you want a dev flow to run all the tests you should define it in the root package.json
and only then consider if to add it to launch.json

Reply:
removed